content/webapp/pages/page.js

```diff
@@ -74,16 +83,28 @@ export class Page extends Component<Props> {
         : 'What we do';
     }
     // TODO: This is not the way to do site sections
+    const sectionItem = page.siteSection
+      ? [
+          {
+            text: getBreadcrumbText(page.siteSection, page.id),
+            url: page.siteSection ? `/${page.siteSection}` : '',
+          },
+        ]
+      : [];
     const breadcrumbs = {
-      items: page.siteSection
-        ? [
-            {
-              text: getBreadcrumbText(page.siteSection, page.id),
-              url: page.siteSection ? `/${page.siteSection}` : '',
-            },
-          ]
-        : [],
+      items: [
+        ...sectionItem,
+        // Only using the first of the siblingsGroup in the list
+        // This should be fine as in reality there shouldn't be more than one
```

Comment:
If this is something we don't expect, why not just treat it as a list and add more, so when it happens, it's not hidden from the UI and we can address it?